tests/lib/hardware/hardware_base.py

```diff
@@ -147,6 +148,23 @@ def execute_ansible_play(self, play_source):
 
         return self._ansible_runner.run_play(play_source)
 
+    def execute_ansible_play(self, play_source):
+        r = self._execute_ansible_play(play_source)
+        failure = False
+        if r.host_failed or r.host_unreachable:
```

Comment:
I guess you want to remove `r.host_failed` ?

Reply:
Woops. Yes, thanks.